src/serenity/config.hpp

```diff
@@ -11,140 +12,133 @@
 #include "serenity/serenity.hpp"
 
 #include "stout/option.hpp"
+#include "stout/result.hpp"
 
 namespace mesos {
 namespace serenity {
 
 /**
- * Global Serenity Config class which implements basic mechanism
- * to support specifying config parameters via string key map.
+ * Serenity Config class which implements basic mechanism
+ * to support specifying config parameters via string key map & sections.
  *
  * Check config_test.cpp to see example usage.
- *
- * TODO(skonefal): every getter should pack result in Try<T>.
  */
 class SerenityConfig {
  public:
   SerenityConfig() {}
 
   /**
-  * Variant type for storing multiple types of data in configuration.
-  */
+   * Variant type for storing multiple types of data in configuration.
+   */
   using CfgVariant = boost::variant<
```

Comment:
How about _Value_? 
Storing 'variant' in name suggest internal structure that backs the field. When we will change the Variant class to something else, do we also want to change name there?